provision/aws/rds/aurora_create.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+set -e
+
+if [[ "$RUNNER_DEBUG" == "1" ]]; then
+  set -x
+fi
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+source ${SCRIPT_DIR}/aurora_common.sh
+
+# https://cloud.redhat.com/blog/using-vpc-peering-to-connect-an-openshift-service-on-an-aws-rosa-cluster-to-an-amazon-rds-mysql-database-in-a-different-vpc
+EXISTING_INSTANCE=$(aws rds describe-db-instances \
+  --query 'DBInstances[*].[DBInstanceIdentifier]'  \
+  --filters Name=db-instance-id,Values=${AURORA_INSTANCE} \
+  --output text
+)
+if [ -n "${EXISTING_INSTANCE}" ]; then
+  echo "Aurora instance '${AURORA_INSTANCE}:${AWS_REGION}' already exists"
+  exit 1
+fi
+
+# Create the Aurora VPC
+AURORA_VPC=$(aws ec2 create-vpc \
+  --cidr-block ${AURORA_VPC_CIDR} \
+  | jq -r '.Vpc.VpcId'
```

Comment:
The `aws` commands which are piping into `jq` should explicitly use `--output json`.